dracut/Dockerfile.in

```diff
@@ -6,9 +6,14 @@
 from opensuse/tumbleweed
 
 RUN zypper --non-interactive refresh; \
-    zypper --non-interactive install --no-recommends --force-resolution cpio curl dracut gcc git-core make tar; \
+    zypper --non-interactive install --no-recommends --force-resolution cpio curl dracut gcc git-core make tar autoconf automake m4 cmake \
```

Comment:
Could you break this list up with backslashes too? (bonus points for sorting the list alphabetically too :)